QRScanner/QRScannerView.swift

```diff
@@ -89,6 +102,9 @@ public class QRScannerView: UIView {
 
     public func rescan() {
         guard isAuthorized() else { return }
+        if blurEffectEnable {
+            blurEffectView.isHidden = true
+        }
```

Comment:
How about remove `if` statement?
I think it is ok that only `blurEffectView.isHidden = true`

Reply:
I see!